department-of-veterans-affairs/next-build
# feat: Issue 18546 next event filtering

## Description
relates to #18456. We noticed that next was filtering out to many events. 

## Testing done
Checked some items from page 1 and page 2. Checked they matched the filter criteria in cms dev. all returned results matched the filters. 

## QA steps
go to /outreach-and-events/events/ . 
pick an event
go to the CMS thats serving the content
login
find the node
verify the event matches the target filter criteria ( the event is publised, and it either exist on outreach hub or has publish to outreach hub checkbox checked)

Example event posted only to outreach hub
![image](https://github.com/user-attachments/assets/d8856b94-fbba-4182-9dbd-7e1f6ce2977c)
![image](https://github.com/user-attachments/assets/f9acd70a-242d-43fe-a4f7-efcb1cf58b03)


Example event posted to daytona health care with checkbox to outreach cal
![image](https://github.com/user-attachments/assets/351974d6-fca3-427d-bea9-b5014005e4f5)
![image](https://github.com/user-attachments/assets/6ccdf6aa-18b1-4aa1-ac31-b5764e53e5ac)


Example of daytona-healthcare with even that is not pubished to outreach cal
![image](https://github.com/user-attachments/assets/21dd86b4-32dd-4c3f-81be-a989d10e091f)
![image](https://github.com/user-attachments/assets/25f8e3db-77fb-4c3a-baa9-c35e3611a846)

Example of daytona-healthcare with even that is published
![image](https://github.com/user-attachments/assets/f488cbd6-40ec-442c-be7d-145ce4e59ac0)
